test/dhcp/dhcp_api_test.rb

```diff
@@ -37,7 +37,26 @@ def test_api_01_get_root
     get "/"
     assert last_response.ok?, "Last response was not ok: #{last_response.status} #{last_response.body}"
     data = JSON.parse(last_response.body)
-    expected = [{"network"=>"192.168.122.0", "netmask"=>"255.255.255.0"}]
+    expected = [{
+        "network"=>"192.168.122.0",
+        "netmask"=>"255.255.255.0",
+        "options"=>{"routers"=>["192.168.122.250"]}
+      },{
+        "network"=>"192.168.123.0",
+        "netmask"=>"255.255.255.192",
+        "options"=>{
+          "routers"=>["192.168.123.1"],
+          "domain_name_servers"=>["192.168.123.1"],
+          "range"=>["192.168.123.2", "192.168.123.62"]
+        }
+      },{
+        "network"=>"192.168.124.0",
+        "netmask"=>"255.255.255.0",
+        "options"=>{
+          "routers"=>["192.168.124.1", "192.168.124.2"],
+          "domain_name_servers"=>["192.168.123.1", "192.168.122.250"]
+        }
+    }]
     assert_equal expected, data
```

Comment:
Test failure here is due to order of entries in `epxected` array being different from the order of entries in `data`. I would suggest using sets for equality comparison (we don't care about the order). You can find an example in other tests in this suite.